test/runnable/extra-files/printVersionUnittest.d

```diff
@@ -0,0 +1,6 @@
+module printVersionUnittest;
```

Comment:
If the next module is `printingUnittest`, this one should be `pragmaMsgVersionUnittest`, no?

Reply:
meh, not a big deal, kind of a long name